packages/components/src/utils/useContainerQueries.tsx

```diff
@@ -0,0 +1,114 @@
+import type React from 'react'
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { useDebouncedCallback } from 'use-debounce'
+
+const DEFAULT_DEBOUNCE_MS = 1000
+
+/**
+ * Tailwind CSS default container query breakpoints in pixels
+ * These match the default values from @tailwindcss/container-queries plugin
+ */
+const DEFAULT_BREAKPOINTS = {
```

Comment:
This is the same as tailwind container [query ranges](https://tailwindcss.com/docs/responsive-design#container-size-reference)

Reply:
Can these be imported from the tailwind package rather than re-creating them?

Reply:
I dont think so but let me check